### PR TITLE
doc: fix spelling errors

### PR DESCRIPTION
**Pull request by fboudra**
Signed-off-by: Fathi Boudra fathi.boudra@linaro.org

### CODE REVIEW

File: src/pn532/pn532.hpp [hunk 1]
```diff
@@ -241,7 +241,7 @@ namespace upm {
     /**
      * configures the SAM (Secure Access Module)
      *
-     * @return true if successfully configured
```
Comment (jontrulson) [inline src/pn532/pn532.hpp]: This one is actually spelled correctly.

File: src/sx1276/sx1276.hpp [hunk 2]
```diff
@@ -126,7 +126,7 @@ namespace upm {
      * same as they were last set during the last RX_DONE event.
      */
     typedef enum {
-      REVENT_DONE                       = 0, // operation completed successfully
```
Comment (jontrulson) [inline src/sx1276/sx1276.hpp]: This one is correct too.

File: src/sx1276/sx1276.hpp [hunk 3]
```diff
@@ -1807,8 +1807,8 @@ namespace upm {
 
     /**
      * Start a receive operation.  The method will return when
-     * completed, either successfully, or in error (crc, or other
-     * issue).  If completed successfully, the returned buffer can be
```
Comment (jontrulson) [inline src/sx1276/sx1276.hpp]: As are these.

**Comment by fboudra** [comment]
my bad, sed mistake. I thought I filtered out successfully replacement.

**Comment by jontrulson** [comment]
Merged, thanks!

**Pull request merged**